### DISCUSSION TITLE
"İşlem sonrasında benzin ücretlerini görmek ister misiniz?"

### DISCUSSION THREAD

**Discussion by kahkaha54** [zkSync-Community-Hub/zksync-developers#358] [Polls]
"Would you like to see how much gas you paid on zkSync compared to other platforms after completing a transaction?"

**Comment by Ea1154** [comment]
nice idea

**Comment by berronesjosh** [comment]
Yes, providing a gas cost comparison feature on zkSync would enhance transparency and help users and developers make informed decisions about transaction costs.

**Comment by renemilnev** [comment]
Sure! With this feature, users can easily track their gas usage on zkSync compared to other platforms, providing valuable insights into transaction costs and efficiency.

**Comment by deanhobson** [comment]
The feature being described allows users to compare the amount of gas they paid on zkSync to other platforms after completing a transaction. This provides users with valuable information and insights into their gas usage, allowing them to make more informed decisions about which platforms to use for their transactions.
